WordPress/src/main/java/org/wordpress/android/ui/posts/PrepublishingViewModel.kt

```diff
@@ -67,7 +68,9 @@ class PrepublishingViewModel @Inject constructor(private val dispatcher: Dispatc
     }
 
     private fun navigateToScreen(prepublishingScreen: PrepublishingScreen) {
-        _dismissKeyboard.postValue(Event(Unit))
+        if (prepublishingScreen != TAGS && prepublishingScreen != HOME) {
```

Comment:
Do we want to put a comment here as to why we don't want to do dismissal for these two screens? Also would this be clearer if we just use `prepublishingScreen != PUBLISH` ? LMK 🙏

Reply:
I understand the simplification that `prepublishingScreen != PUBLISH` would bring, however I think it's better to be explicit in the case we in the future add other screens that don't need the dismiss event to be sent. The problem here is more about navigating from/ to Home / Tags which have EditText. Probably we should flag them in some other meaningful way (as it looks arbitrary otherwise), but it starts making the workaround more convoluted.
As I keep thinking about it, I realize a comment in this case would suffice for the most part, so I'll add the suggested optimization and the comment 👍

Reply:
Addressed in fd0019fb4f 👍

Reply:
Awesome. Thanks @mzorz for this explanation! The comment makes everything clear now and thanks for implementing my solution. You really didn't have to because your explanation makes sense😅 but I truly appreciate it 😄